lib/config.js

```diff
@@ -38,6 +38,10 @@ var Config = function(src, options) {
       }
     }, this);
   }
+
+  if (options.public) {
```

Comment:
Since `--public` isn't a global option I'd prefer not to have the config loader do the check here. It probably belongs in `deploy/hosting/prepare.js`.